docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -295,41 +295,63 @@ V&V Checks:
     - GBD
     - Sum of GBD ectopic and abortion and miscarriage rates
 
+Unlike the pregnancy model (for which we used state tables for V&V), we are requesting count data from the observers, as 
+outlined in the table below. 
+
 
 .. list-table:: Outputs from Intrapartum Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
-  * - Input
-    - Data Source 
-    - Notes
-  * - Delivery facility type
+  * - Output
+    - Data Source
+    - Stratifications
+    - Use case
+  * - Counts of simulants attending each delivery facility type
+      (BeMONC, CeMONC, home)
     - Decision tree point
-    - 
-  * - Interventions received (c-section, azithromycin, corticosteroids)
-    - Decision tree values
-    - 
-  * - Count of maternal disorders
-    - Simulant experiences in model
-    - 
-  * - Maternal outcomes
-    - Simulant experiences in model
-    - To be defined, YLLs, YLDs, deaths, etc. 
-  * - Type of birth
-    - Simulant experiences in model
-    - E.g., live, still 
+    - ?
+    - Input for later modeling stage & export for V&V
```

Comment:
I think the input for later modeling stages will need to be a state table.   Maybe this should be a table of exclusively the things needed for V&V.